pkgs/development/libraries/science/math/tiny-cuda-nn/default.nix

```
@@ -157,5 +157,7 @@ in
       license = licenses.bsd3;
       maintainers = with maintainers; [connorbaker];
       platforms = platforms.linux;
+      # g++: error: unrecognized command-line option '-mf16c'
+      broken = stdenv.hostPlatform.system == "aarch64-linux";
```

Is there a way to do this without string comparisons? Maybe something like (I wrote this in a GitHub comment so I've not tested it):

```suggestion
      broken = 
        let
          inherit (stdenv.hostPlatform) isAarch isLinux;
        in
        isAarch && isLinux;
```

---

I went with @Aleksanaa's solution.